packages/dappmanager/src/modules/stakerConfig/setStakerConfig.ts

```diff
@@ -508,3 +520,70 @@ function setStakerConfigOnDb<T extends Network>({
       throw new Error(`Unsupported network: ${network}`);
   }
 }
+
+function checkLodestarMinVersions(
+  lodestarDnpName: string,
+  selectedExecClient: StakerItemOk<Network, "execution"> | undefined,
+  isSignerSelected: boolean | undefined,
+  web3signerPkg: InstalledPackageDataApiReturn | undefined
+): void {
+  if (!selectedExecClient) {
+    throw Error(`Execution client is not selected.`);
```

Comment:
This is check is wrong, the user may unselect the execution client which means that this value will be empty/undefined

Reply:
But it should only enter the function if a CC is selected (lodestar) and this means that you also need a EC, right @pablomendezroyo ?

Reply:
Then the function should be called only if both clients exists and you do not have to re-check it inside the function